pyproject.toml

```diff
@@ -75,7 +75,7 @@ exporters = [
     "am2r-yams>=2.8.1",
 
     # Cave Story
-    "cave-story-randomizer>=2.3.3",
+    "cave-story-randomizer>=2.4.2",
```

Comment:
changelog?

Reply:
we currently use 2.4.1 (see requirements.txt). the only change is the typing backend which doesn't affect users

Reply:
should this not create a conflict then if we're currently using 2.4.1? :thinking:

Reply:
I mean... it doesn't? so who cares? why would this create a conflict, anyway?

Reply:
I assumed this line was 2.4.1 on main, but shows 2.3.3 on the diff, thus should lead to a conflict.
But it's not this line that's 2.4.1 on main, it's the requirements.txt as you pointed out.

It doesn't matter, was just trying to understand it for myself.